cmd/eudico/daemon.go

```diff
@@ -193,6 +194,20 @@ func daemonCmd(overrides node.Option) *cli.Command {
 			// Set the metric to one so it is published to the exporter
 			stats.Record(ctx, metrics.LotusInfo.M(1))
 
+			go func() {
```

Comment:
We should probably extract this into its own command. We could start this in an independent process, spawn the required APIs from the repo, and spawn all the metrics-related processes that we need.

Either that or this should be opt-in through an env variable.

Reply:
I'll add an env variable for now.